Traceback (most recent call last):
  File "/well/win-fmrib-analysis/users/gqu790/moval/moval_scripts/estim_seg3d.py", line 104, in <module>
    main()
  File "/well/win-fmrib-analysis/users/gqu790/moval/moval_scripts/estim_seg3d.py", line 94, in main
    moval_model.fit(logits, gt)
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/moval/integrations/sklearn/moval.py", line 236, in fit
    solver.fit(logits, gt, batch) # model fitting
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/moval/solvers/solver.py", line 300, in fit
    optimization_result = scipy.optimize.minimize(
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/scipy/optimize/_minimize.py", line 684, in minimize
    res = _minimize_neldermead(fun, x0, args, callback, bounds=bounds,
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/scipy/optimize/_optimize.py", line 845, in _minimize_neldermead
    fsim[k] = func(sim[k])
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/scipy/optimize/_optimize.py", line 569, in function_wrapper
    fx = function(np.copy(x), *(wrapper_args + args))
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/moval/solvers/solver.py", line 82, in eval_func
    estim_perf = self.model.estimate_precision(probability, gt_guide = self.gt_guide)
  File "/well/win-fmrib-analysis/users/gqu790/conda/skylake/envs/moval/lib/python3.8/site-packages/moval/models/model.py", line 296, in estimate_precision
    raise ValueError(f"estimation algorithm '{self.estim_algorithm}' does not support the estimation of precision (because FP is always 0)")
ValueError: estimation algorithm 'ts-atc-model' does not support the estimation of precision (because FP is always 0)
